app/src/components/TransactionStatus.vue

```diff
@@ -47,15 +58,14 @@ import useWalletStore from 'src/store/wallet';
 const emittedEventName = 'onReceipt'; // emitted once we receive the transaction receipt
 const rowClasses = [
   // styles applied to all rows in the HTML template
-  'flex',
-  'justify-start',
-  'items-center',
   'text-grey-400',
   'text-left',
   'py-10',
   'px-4',
   'sm:px-6',
   'lg:px-8',
+  'grid',
+  'grid-cols-12',
```

Comment:
Nice grid use, I really need to learn grids one day instead of forcing everything into flexbox haha 🙃